draft-ietf-taps-interface.md

```diff
@@ -1554,66 +1554,98 @@ Security parameters and callbacks are partitioned based on their place in the li
 of Connection establishment. Similar to Transport Properties, both parameters and callbacks
 are inherited during cloning (see {{groups}}).
 
-### Specifying Security Parameters on a Pre-Connection
-
-Common security parameters such as TLS ciphersuites are known to implementations. Clients should
-use common safe defaults for these values whenever possible. However, as discussed in
-{{?RFC8922}}, many transport security protocols require specific
+This document specifies an abstract API, which may appear to conflict with the need
+for security parameters to be unambiguous in order to reduce the chances
+of implementation errors. The Transport Services system SHOULD provide reasonable,
+secure defaults for each enumerated security parameter, such that users of the system
+only need to specify parameters required to establish a secure connection
+('server-certificate', 'client-certificate'). Specifying specific security parameters
+from enumerated values (e.g., specific ciphersuites) may constrain the Transport
+Protocols that can be selected during connection establishment.
+
+Except as noted below, as with the rest of the Transport Services API, exact names of parameters and/or
+values of enumerations (e.g., ciphersuites) used in the security parameters are system
+and implementation specific, and should be chosen to follow the principle of least
+surprise for users of the platform / language environment in question.
+
+### Specifying Security Parameters on a Preconnection
+
+Common security parameters such as TLS ciphersuites are known to implementations.
+Clients should use common safe defaults for these values whenever possible.
+However, as discussed in {{?RFC8922}}, many transport security protocols require specific
```

Comment:
/Clients/ - is that applictaion, the APi, the transport services system?  I was not sure.

Reply:
it's the app, editing.